runner/src/server/plugins/blankie.ts

```diff
@@ -11,15 +11,20 @@ export const configureBlankiePlugin = (
   return {
     plugin: Blankie,
     options: {
-      fontSrc: ["self", "data:"],
-      scriptSrc: (() =>
-        ["self", "unsafe-inline"].concat(
+      connectSrc: (() =>
+        ['self'].concat(
           config.matomoUrl ? [config.matomoUrl] : []
         ))(),
-      styleSrc: ["self", "unsafe-inline"],
+      defaultSrc: ['self'],
+      fontSrc: ['self', 'data:'],
+      scriptSrc: (() =>
+        ['self', 'unsafe-inline'].concat(
+          config.matomoUrl ? [config.matomoUrl + '/piwik/piwik.js'] : []
+        ))(),
+      styleSrc: ['self', 'unsafe-inline'],
       imgSrc: (() =>
-        ["self"].concat(config.matomoUrl ? [config.matomoUrl] : []))(),
-      generateNonces: false,
+        ['self'].concat(config.matomoUrl ? [config.matomoUrl] : []))(),
+      generateNonces: false
```

Comment:
Hi Trevor, thanks for the PR! 

It looks like you aren't running prettier and/or your git client is bypassing the git hook which applies styling. Please rerun, the obvious ones I can see are it should be `"` and trailing comma in object definitions.